Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -168,7 +168,7 @@ internal set
         //  suppress "Field 'NetworkIdentity.m_SceneId' is never assigned to, and will always have its default value 0"
         // when building standalone
 #pragma warning disable CS0649
-        [SerializeField] ulong m_SceneId;
+        [SerializeField, HideInInspector] ulong m_SceneId;
```

Comment:
Why should we even keep this since we have a public field on line 96?
```cs
public ulong sceneId => m_SceneId;
```
Just get rid of `m_SceneId` in favor of `sceneId` ?

Reply:
@MrGadget1024 we should do that in a separate PR. one small step after another.